content/docs/command-reference/add.md

```diff
@@ -148,6 +171,20 @@ not.
   > Note that external outputs typically require an external cache setup. See
   > link above for more details.
 
+- `--to-remote` - add target data into DVC and create a .dvc file, but instead
+  of caching it into DVC cache, transfer it straight to remote storage. Check
+  [this](#transferring-data-directly-to-the-remote) section for the details. If
+  this option is specified target can be any cloud or local URL, not necessarily
+  a local file or directory from the workspace as it is required in the regular
+  dvc addworkflow.
```

Comment:
```suggestion
- `--to-remote` - track a single external target file or directory (with a `.dvc` file),
  but instead of caching and linking it locally,
  [transfer](#transferring-data-directly-to-the-remote) it straight to remote
  storage.
```

Reply:
Should we update https://dvc.org/doc/user-guide/managing-external-data too and link from here, like it's linked from `--external`?